app/templates/content.hbs

```diff
@@ -79,7 +79,17 @@
                     </div>
                     <div class="p-t-xs">
                         <h4 class="p-v-md f-w-md"><strong>{{t "global.abstract"}}</strong></h4>
-                        <p class="abstract">{{node.description}}</p>
+                        {{#if useShortenedDescription}}
+                            {{#if expandedAbstract}}
+                                <p class="abstract">{{node.description}}</p>
+                                <a href='#' {{action 'expandAbstract'}}>See less</a>
```

Comment:
Accessibility note: we try to avoid using `<a>` tags for interactive elements not related to navigation.

To better communicate the intent of this element to screen readers, consider a `<button class="btn-link">` instead.